pkg/cmd/taskrun/logs.go

```diff
@@ -39,6 +41,45 @@ type LogOptions struct {
 	AllSteps bool
 }
 
+func logCommand(p cli.Params) *cobra.Command {
+	opts := LogOptions{}
+	eg := `
+	# show the logs of TaskRun named "foo" from the namesspace "bar"
```

Comment:
doesn't this add unwanted space in the beginning?

Reply:
Yes, it will. Removed unwanted space.